tests/test_cmdline.py

```diff
@@ -164,8 +164,22 @@ async def print_sum(x, y):
 
 class TestCommandLineBasic(CmdlineTmpl):
     def test_no_file(self):
-        result = self.template(["python", "-m", "viztracer"], expected_output_file=None)
-        self.assertIn("help", result.stdout.decode("utf8"))
+        result = self.template(["python", "-m", "viztracer"], expected_output_file=None).stdout.decode("utf8")
+        self.assertIn("help", result)
+        #  Check for a few more arguments to ensure we hit the intended argumentParser
+        self.assertIn("--output_file", result)
+        self.assertIn("--log_async", result)
+        self.assertIn("--attach", result)
+        self.assertIn("--plugins", result)
+        self.assertIn("--rcfile", result)
+
+    def test_help(self):
+        """Test that all three options print the same help page"""
+        result = self.template(["python", "-m", "viztracer"], expected_output_file=None).stdout.decode("utf-8")
```

Comment:
As I mentioned above, I would rather not to use ``result`` as the variable to store the stdout string in this file. You can either get the stdout data from the result, or use chain and a different variable name than ``result``.